sympy/physics/control/control_plots.py

```diff
@@ -863,18 +865,35 @@ def bode_phase_numerical_data(system, initial_exp=-5, final_exp=5, freq_unit='ra
     x, y = LineOver1DRangeSeries(phase,
         (_w, 10**initial_exp, 10**final_exp), xscale='log', **kwargs).get_points()
 
+    if phase_unwrap:
+        if(phase_unit == 'rad'):
+            for i in range(1, len(y)):
+                diff = y[i] - y[i - 1]
+                if diff > pi:      # Jump from -pi to pi
+                    y[i] = (y[i] - 2*pi)
+                elif diff < -pi:   # Jump from pi to -pi
+                    y[i] = (y[i] + 2*pi)
+
+        elif(phase_unit == 'deg'):
+            for i in range(1, len(y)):
+                diff = y[i] - y[i - 1]
+                if diff > 180:      # Jump from -180 to 180
+                    y[i] = (y[i] - 360)
+                elif diff < -180:   # Jump from 180 to -180
+                    y[i] = (y[i] + 360)
```

Comment:
```suggestion
        half = None
        if(phase_unit == 'rad'):
            half = pi
        elif(phase_unit == 'deg'):
            half = 180
        if half:
            unit = 2*half
            for i in range(1, len(y)):
                diff = y[i] - y[i - 1]
                if diff > half:      # Jump from -half to half
                    y[i] = (y[i] - unit)
                elif diff < -half:   # Jump from half to -half
                    y[i] = (y[i] + unit)
```

Reply:
I get what you're trying to refactor. Making a small modification so that we do not unwrap it when user gives `phase_unwrap = false`.

```
+        half = None
+        if phase_unwrap:
+            if(phase_unit == 'rad'):
+                half = pi
+            elif(phase_unit == 'deg'):
+                half = 180
+        if half:
+            unit = 2*half
+                if diff > half:      # Jump from -half to half
+                    y[i] = (y[i] - unit)
+                elif diff < -half:   # Jump from half to -half
+                    y[i] = (y[i] + unit)
```